test/check-cli

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/python3
+
+import composertest
+
+
+class TestCli(composertest.ComposerTestCase):
+    def runCliTest(self, script):
+        r = self.execute(["CLI=/usr/bin/composer-cli", "/tests/test_cli.sh", script])
+        self.assertEqual(r.returncode, 0)
+
+    def testBlueprintsSanity(self):
+        self.runCliTest("/tests/cli/test_blueprints_sanity.sh")
+
+    def testComposeSanity(self):
+        self.runCliTest("/tests/cli/test_compose_sanity.sh")
+
+
+class TestImages(TestCli):
```

Comment:
This will also inherit the `test*Sanity` methods and execute them when this class is executed, which will add more time to the execution duration. Probably not what you want.

IMO move `runCliTest()` into `ComposerTestCase` class.

Reply:
Oops indeed. Changed it back. Thanks!